Fixing v1.24 windows jobs by updates CAPZ version used to build the test clusters

marosset opened the pull request:
Fixing v1.24 SIG-Windows e2e jobs

Most of the other jobs are still on capz@release-1.8 so I moved the 1.24 jobs to release-1.8 too and we can then move all the windows jobs to capz@release-1.10 together

/assign @jsturtevant 
/sig windows

jsturtevant commented:
> Most of the other jobs are still on capz@release-1.8 so I moved the 1.24 jobs to release-1.8 too and we can then move all the windows jobs to capz@release-1.10 together

I think we need 1.9+ to fix the golang bug.  1.25+ is running 1.10 https://github.com/kubernetes/test-infra/blob/032f9e46e570729063f142f09905236661ff81eb/config/jobs/kubernetes-sigs/sig-windows/release-1.25-windows.yaml#L34-L35

marosset commented:
> > Most of the other jobs are still on capz@release-1.8 so I moved the 1.24 jobs to release-1.8 too and we can then move all the windows jobs to capz@release-1.10 together
> 
> I think we need 1.9+ to fix the golang bug. 1.25+ is running 1.10
> 
> https://github.com/kubernetes/test-infra/blob/032f9e46e570729063f142f09905236661ff81eb/config/jobs/kubernetes-sigs/sig-windows/release-1.25-windows.yaml#L34-L35

oops, my enlistment must have been stale

marosset commented:
> > > Most of the other jobs are still on capz@release-1.8 so I moved the 1.24 jobs to release-1.8 too and we can then move all the windows jobs to capz@release-1.10 together
> > 
> > 
> > I think we need 1.9+ to fix the golang bug. 1.25+ is running 1.10
> > https://github.com/kubernetes/test-infra/blob/032f9e46e570729063f142f09905236661ff81eb/config/jobs/kubernetes-sigs/sig-windows/release-1.25-windows.yaml#L34-L35
> 
> oops, my enlistment must have been stale

updated!

jsturtevant commented:
/lgtm

jsturtevant commented:
/lgtm
